recipes/scikit-build/python27-x64.patch

```diff
@@ -0,0 +1,13 @@
+MSVC 9 x64 is broken on appveyor. Use NMake there.
```

Comment:
> requires admin privileges.

I think in practice it is reasonable to assume developers have admin privileges on their workstation. 

Similarly to the installation of recent Visual Studio (e.g VS2015 for Python3.5) already require admin privileges, expecting developer to patch their install of Visual Studio 2008 express seems fair.

That said, installing arbitrary patches to modify an official installation could be a problem here and supporting " Microsoft Visual C++ Compiler for Python 2.7" would be a more sane alternative.